docs/theory_manual/theory.rst

```diff
@@ -402,15 +402,62 @@ Then other magnitudes such as the Froude-Krylov forces or the added mass can be
 Post-processing
 ===============
 
-Forces on the body
-------------------
+Forces on body surfaces
+-----------------------
 
-TODO
+Forces acting on body surfaces are computed by integration of the pressure field. They can be decomposed into three contributions:
+
+1. The Froude-Krylov forces :math:`F_{FK, i}`, from the integration of the incident wave field pressure (incoming plane waves); :math:`i` denotes the i-th degree of freedom
+2. The diffraction forces :math:`F_{D, i}`, from the integration of the diffracted wave field (all bodies held fixed)
+3. The radiation forces :math:`F_{R, ij}`, from the result of the radiation problem with radiating degree of freedom :math:`j` and influenced degree of freedom :math:`i`
+
+Dynamic coupling and impedance
+------------------------------
+Consider a body or a system of bodies. The general linear equation of motion can be expressed in time domain as 
+
+.. math:: M_{ij} \ddot{x}_j + C_{ij} \dot{x}_j + K_{ij} x_j = F_i,
+
+and in frequency domain, with the assumed time dependence :math:`x(t) = \mathrm{Re} \left( X e^{-j \omega t} \right)`,
+
+.. math:: \left[-\omega^2M_{ij} - j \omega C_{ij} + K_{ij}\right] X_j = F_i,
+
+where :math:`M_{ij}` is the inertia matrix, accounting for the mass distribution, :math:`C_{ij}` is the mechanical damping matrix, :math:`K_{ij}` is the stiffness matrix which comprises mechanical and hydrostatic effects, and :math:`F_i` are generic external forces.
+
+.. note:: The hydrostatic contribution to matrix :math:`K_{ij}` accounts for a variation of hydrostatic force in direction :math:`i` due to a unit motion in direction :math:`j`. It is a geometric property of the body.
+
+Forces :math:`F_i` can be decomposed as
+
+.. math:: F_i = F_{FK, i} + F_{D, i} + F_{R, ij}
+
+and :math:`F_{R, ij}` can be further rewritten as 
+
+.. math:: F_{R, ij} = \left[\omega^2 A_{ij} + j\omega B_{ij}\right] X_j
+
+where :math:`A_{ij}` is the added mass matrix and :math:`B_{ij}` is the radiation damping matrix; these properties are thus obtained from the real and imaginary parts of the radiation force. The full system becomes
+
+.. math:: \left[-\omega^2 (M_{ij} + A_{ij}) - j \omega (C_{ij} + B_{ij}) + K_{ij}\right] X_j = F_i
+
+and the oscillation amplitude is obtained by solving the system.
+
+.. note:: Matrices :math:`A_{ij}` and :math:`B_{ij}` depend on :math:`\omega`, and so does :math:`X_j`.
 
 Free surface elevation
 ----------------------
 
-TODO
+The potential at the reference surface :math:`z = 0` can be connected to the free surface elevation by the dynamic condition
+
+.. math:: \dfrac{\partial \phi}{\partial t} = - g \eta
+
+which, in frequency domain, is
+
+.. math:: \eta = \dfrac{j \omega}{g} \Phi
+
+For a fully coupled problem (bodies free to oscillate, i.e. diffraction and radiation combined), the free surface elevation can be computed as 
+
+.. math:: \eta = \eta_{\text{incident}} + \eta_{\text{diffracted}} -j \omega \sum_i \eta_{\text{radiated}, i}   X_i
```

Comment:
Wait. Shouldn't there be also a `-jω` in front of the diffraction term? Isn't the diffraction problem solved with the same conventions?

Reply:
I see what you mean, the `jω` of the diffraction is already in `airy_wave_velocity`.

Actually, it might make sense to add a `jω` directly in https://github.com/mancellin/capytaine/blob/de208c89ddbe5592253128166c32436cfcd8c68a/capytaine/bem/problems_and_results.py#L283 to have the same conventions everywhere. The only problem of doing that is that it would confuse everybody used to the current implementation...

Reply:
Yes, that would mean having to correct the computation of added mass and radiation damping accordingly.